lib/ui/page/home/page/chat/controller.dart

```diff
@@ -611,10 +622,35 @@ class ChatController extends GetxController {
     }
 
     super.onReady();
+
+    _restorePreviousScroll();
+  }
+
+  /// Restores the scroll position of the current chat after the UI frame is built.
+  /// This method retrieves the previously saved [ChatScrollInfo] using a unique
+  /// key composed of the current user ID (`me!.val`) and chat ID (`id.val`).
+  /// It then scrolls the list to the stored index and offset using [jumpToIndex],
+  /// optionally using offset from the bottom if specified.
+  void _restorePreviousScroll() {
```

Comment:
This function is a private function and it should be positioned after any public methods according to the [Contribution Guide](https://github.com/team113/messenger/blob/main/CONTRIBUTING.md#classes-constructors-fields-and-methods-ordering).